app/views/layouts/application.html.erb

```diff
@@ -3,10 +3,15 @@
   <head>
     <title>WatchTheSkiesDashboard</title>
     <%= stylesheet_link_tag    'application', media: 'all'%>
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.3/angular.js"></script>
```

Comment:
Note to self: Figure out why Bower stopped playing nicely with Angular. 
It's probably fine to use this CDN approach, but if that's the case, let's rip the logic out elsewhere.